src/components/Movies/Movies.js

```diff
@@ -1,16 +1,20 @@
 import "./Movies.scss";
 import MovieCard from "../MovieCard/MovieCard";
+import StarRating from '../StarRating/StarRating'
 
 const Movies = ({ apiMovieData }) => {
   const movieCards = apiMovieData.map(movie => {
     return (
+      <div>
       <MovieCard
         key={movie.id}
         id={movie.id}
         title={movie.title}
         poster={movie.poster_path}
         rating={movie.average_rating}
       />
+      <StarRating rating={Math.round(movie.average_rating)} />
```

Comment:
Ah! Okay, so this is how you went about this because I had a different idea, but we can discuss what you did.